src/main/resources/admin/widgets/archive/archive.html

```diff
@@ -0,0 +1,3 @@
+<widget id="widget-archive">
+    <script src="{{assetsUri}}" configServiceUrl="{{configServiceUrl}}" i18nServiceUrl="{{i18nServiceUrl}}" elemId="widget-archive"></script>
```

Comment:
Same changes as for the Settings widget: change attribute names to be all-lower-case data attributes. Btw, do you really need to pass `i18nServiceUrl` as an attribute? It should be returned by the config service.